documentation/source/welcome.md

```diff
@@ -38,442 +12,81 @@ ______________________________________________________________________
 Welcome to SuperGradients, a free, open-source training library for PyTorch-based deep learning models.
 SuperGradients allows you to train or fine-tune SOTA pre-trained models for all the most commonly applied computer vision tasks with just one training library. We currently support object detection, image classification and semantic segmentation for videos and images.
 
-Docs and full user guide[](#)
-### Why use SuperGradients?
+
+## Why use SuperGradients?
  
-**Built-in SOTA Models**
+### Built-in SOTA Models
 
 Easily load and fine-tune production-ready, [pre-trained SOTA models](https://github.com/Deci-AI/super-gradients#pretrained-classification-pytorch-checkpoints) that incorporate best practices and validated hyper-parameters for achieving best-in-class accuracy.
```

Comment:
I also think it would be great to name drop a little bit to seem attractive (ppyoloe, yolox, PP-LiteSeg for instance?).
Because I feel like people might think this is just another repo that refers to SOTA to actually refer to old models.

Reply:
fixed both